pkg/urbit/jets/d/in_int.c

```diff
@@ -3,74 +3,56 @@
 */
 #include "all.h"
 
-
 /* functions
 */
-  u3_noun
-  u3qdi_int(u3_noun a,
-            u3_noun b)
-  {
-    if ( u3_nul == a ) {
-      return u3k(u3_nul);
+u3_noun
+u3qdi_int(u3_noun a,
+          u3_noun b)
+{
+  if ( u3_nul == a ) {
+    return u3k(u3_nul);
+  }
+  else if ( u3_nul == b ) {
+    return u3k(u3_nul);
+  }
+  else {
+    u3_noun l_a, n_a, r_a, lr_a;
+    u3_noun l_b, n_b, r_b, lr_b;
+    u3_noun c;
+    u3x_cell(a, &n_a, &lr_a);
+    u3r_cell(b, &n_b, &lr_b);
```

Comment:
Should this second one also be a u3x instead of u3r?

Reply:
Good catch, thanks.